spec/controllers/conferences_controller_spec.rb

```diff
@@ -54,4 +54,58 @@
       end
     end
   end
+
+  describe 'POST create' do
+    context 'with an unauthenticated user' do
+      let(:params) { attributes_for :conference }
```

Comment:
Minor naming thing, but technically speaking these aren't the "params"... How about `conference_attrs`?

Reply:
Hmm.. I don't understand the difference. If I print params, I get a hash with name, url, etc, just like the post parameters.

Reply:
Sorry, it's just a naming thing 😉 
The "params" look like this: `{ conference: { name: "bla", url: "yay.de" ... } }`
What you assign as params here is only the `{ name: "bla", etc }` part.

But again, this is only a question of naming things, has nothing to do with the test itself. The code work fine.